lhotse/recipes/reazonspeech.py

```diff
@@ -0,0 +1,69 @@
+import json
+import logging
+from collections import defaultdict
+from pathlib import Path
+from typing import Dict, Optional, Union
+
+from lhotse import validate_recordings_and_supervisions
+from lhotse.audio import Recording, RecordingSet
+from lhotse.qa import fix_manifests
+from lhotse.supervision import SupervisionSegment, SupervisionSet
+from lhotse.utils import Pathlike
+
+
+def prepare_reazonspeech(
```

Comment:
Since reazonspeech is a large dataset this method may require a lot of cpu memory before it writes anything. Would you consider modifying the recipe to resemble gigaspeech more closely? It writes examples as it processes them for reduced memory usage https://github.com/lhotse-speech/lhotse/blob/master/lhotse/recipes/gigaspeech.py